lib/evss/disability_compensation_form/form4142_processor.rb

```diff
@@ -88,8 +88,7 @@ def generate_metadata(jid)
       end
 
       def received_date
-        date = SavedClaim::DisabilityCompensation.find(@submission.saved_claim_id).created_at
-        date = date.in_time_zone('Central Time (US & Canada)')
+        date = @submission.created_at.in_time_zone('Central Time (US & Canada)')
```

Comment:
Just a note that this update is for a 4142 submitted as a part of the default 526 submit process path. From app/controllers/v0/disability_compensation_forms_controller.rb:51, we can see that the `saved_claim` is created at the same time as the `submission` itself, but using the `submission.created_at` timestamp makes more sense here semantically.